lib/playground.dart

```diff
@@ -297,6 +294,46 @@ class Playground {
     querySelector("#consoletab").setAttribute('selected','');
   }
 
+
+  _handleAutoCompletion(KeyboardEvent e) {
+    // If we're already in completion bail.
+    if (_isCompletionActive) return;
+
+    if (context.focusedEditor == 'dart') {
+      if (e.keyCode == KeyCode.PERIOD) {
+        editor.completionAutoInvoked = true;
+        editor.execCommand("autocomplete");
+        new Timer(const Duration(milliseconds: 500), _handleHelp);
```

Comment:
@devoncarew I think something like this could work in editor_codemirror.dart

```dart
      Timer.run((){
        // send a signal here
      });
      return new HintResults.fromHints(hints, from, to);
```

Maybe I could make the handlehelp thing its own class and then import that in playground and editor_codemirror.dart.